src/js/components/TableComponent.jsx

```diff
@@ -0,0 +1,33 @@
+import {DataTable} from 'safe-framework'
+import titleCase from 'title-case'
+import React, {Component, PropTypes} from 'react'
+
+export default class TableComponent extends Component {
+  static propTypes = {
+    data: PropTypes.array.isRequired,
+    type: PropTypes.string.isRequired
+  }
+
+  render () {
+    const {data} = this.props
+    const [firstItem = {}] = data
+
+    const columns = Object.keys(firstItem)
+      .filter((field) => !field.startsWith('_'))
```

Comment:
since we are using more and more ES6 features, we need to do `import 'babel-polyfill'` at the top of `src/js/index.jsx`